internal/cloudrunci/gcloud.go

```diff
@@ -42,8 +42,13 @@ func gcloud(label string, cmd *exec.Cmd) ([]byte, error) {
 	var out []byte
 	var err error
 
+	delaySeconds := 2 * time.Second
+	if strings.Contains(label, "build container image") {
```

Comment:
Where does this label come from? Seems like it could easily change?

Reply:
This is set in the cloudrunci Build function which is used to create all gcloud commands for gcloud builds submit. I think it's safe to use, though I'm happy to create a constant. It is simpler than parsing the command arguments.